packages/io-devices/README.md

```diff
@@ -0,0 +1,58 @@
+# `@wasmer/wasmfs-io-devices`
+
+Isomorphic library to implement the experimental Wasmer I/O devices for wasmfs instances 🖼️🎮
+
+## Table of Contents
+
+- [Features](#features)
+- [Installation](#installation)
+- [Quick Start](#quick-start)
+- [Documentation](#documentation)
+- [Contributing](#contributing)
+
+## Features
+
+This package provides the following features:
+
+- Scaffolds files used by the [Wasmer Runtime I/O Devices](https://docs.wasmer.io/runtime/runtime), to allow WASI moduels that depend on these interfaces to run in the browser and Node! 📁
+- Allows readiing the current RGBA FrameBuffer. 🖼️
+- Can send input events to the I/O Devices interface. ⌨️
+
+## Installation
+
+For installing `@wasmer/wasmfs-io-devices`, just run this command in your shell:
+
+```bash
+npm install --save @wasmer/wasmfs-io-devices
+```
+
+## Quick Start
+
+```js
+import { WasmFs } from "@wasmer/wasmfs";
+import { WasmFsIoDevices } from "@wasmer/wasmfs-io-devices";
+
+const wasmFs = new WasmFs();
+const wasmFsIoDevices = new WasmFsIoDevices(wasmFs);
+
+// Fill the framebuffer, for example, here
```

Comment:
Perhaps we are missing an example here?

Reply:
Ah yes! I guess I forgot to update docs and tests 😢